From 1546de56d0f994d77391507406c017b8327deb76 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Tue, 19 Apr 2016 15:28:44 -0500
Subject: [PATCH 1013/5242] MLK-12684-3: Documentation: gpmi-nand: document
 the gpmi-nand compatibility

commit  921d744971608b8aef74691c53f44993bef1bbb7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Document the gpmi-nand compatibility for i.MX6ULL and others

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/mtd/gpmi-nand.txt          |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
index 44fae30..039f8fb 100644
--- a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
+++ b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
@@ -8,8 +8,11 @@ Required properties:
     * imx23
     * imx28
     * imx6q
+    * imx6qp
     * imx6sx
+    * imx6ul
     * imx7d
+    * imx6ull
   - reg : should contain registers location and length for gpmi and bch.
   - reg-names: Should contain the reg names "gpmi-nand" and "bch"
   - interrupts : BCH interrupt number.
-- 
1.7.9.5

